guide/content/helpers/link.slim

```diff
@@ -45,6 +45,26 @@ markdown:
     The keyword arguments automatically add a space between the visually
     hidden and regular text.
 
+== render('/partials/example.*',
+  caption: "Links that open in a new tab",
+  code: govuk_link_to_new_tab) do
+
+  markdown:
+    You can use the `new_tab` parameter to automatically set the `target` and
+    `rel` [attributes as suggested by the design system][0].
+
+    When the link text is passed in as a string argument, 'opens in new tab' is
+    added to the end automatically. No suffix is added when the helper is used
+    in block mode.
```

Comment:
This confused me at first, as I mis-read as describing when the new tab text is added as a string.

Is it worth having a few different examples?

1. `new_tab: true` for the default text
2. `new_tab: "(opens in a new window)" to override the default
3. `new_tab: govuk_visually_hidden("(opens in a new tab)")` to visually-hide it (example could have a preceding line saying "The following links open in a new tab" to follow the Design System guidance?

Could also mention how to turn off the text altogether, but maybe not include an example, as this pattern isn't recommended?!

Reply:
Agreed and implemented.